reddit.js

```diff
@@ -2,4 +2,65 @@ var reddit = require("./data.json")
 
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+// console.log(reddit["data"]);
+
+
+// Print all reddit titles using forEach: 
+var array = reddit.data.children;
+
+
+
+
+array.forEach(function (array) {
```

Comment:
Here, ```array``` is probably not the best choice for your parameter name. Since ```forEach``` is iterating over each *object* in the children array, 'child' or 'post' is probably a more appropriate representation of what's being dealt with here.